package.json

```diff
@@ -62,6 +62,7 @@
         "less": "^3.10.3",
         "less-loader": "^5.0.0",
         "mini-css-extract-plugin": "^0.8.0",
+        "minimist": "^1.2.5",
```

Comment:
You should update the coveralls library which requires this library. Revert these changes and execute `npm update coveralls`